### DISCUSSION TITLE
What is the online version for and does it still work with the latest browsers?

### DISCUSSION THREAD

**Discussion by krystian3w** [sumatrapdfreader/sumatrapdf#4289] [Q&A] [answered]
https://online.sumatrapdfreader.org/

**Comment by kjk** [comment, marked as answer]
It's an experiment and not really great so ignore it.

**Reply by GitHubRulesOK** [reply]
SumatraPDF plug-in still works with at LEAST 3 remaining current browsers (Waterfox 64bit and SeaMonkey 32/64 bit no longer support NPAPI ) 

Order of rank are 1)PaleMoon 2)Basilisk 3) K-Meleon NONE officially support SumatraPDF and SumatraPDF does not support the plug-in

To see Pale Moon and Basilisk in action with recent releases see comment https://github.com/sumatrapdfreader/sumatrapdf/issues/2754#issuecomment-2138270931
